.github/workflows/ci.yml

```diff
@@ -102,7 +101,9 @@ jobs:
             # Package names are g++-* and clang-*, so set this before correcting CXX for Clang
             pkgs="$CXX xsltproc docbook-xsl docbook-xml lcov ccache"
             sudo apt-get -o Acquire::Retries=$NET_RETRY_COUNT install -y $pkgs
-            CXX="ccache ${CXX/clang-/clang++-}"
+            if [[ "$CXX" =~ clang ]]; then
```

Comment:
Why this change? It disables ccache for GCC

Reply:
I’ll try to put it back, but it was failing saying that the gcc binary was not found it path.

Reply:
Right, that was probably something else, I removed this change.